Traceback (most recent call last):
  File "/home/x/Escritorio/VueCrud/main.py", line 1, in <module>
    from pydantic import BaseModel
ModuleNotFoundError: No module named 'pydantic'
